automation/vars/uni04delta.yaml

```diff
@@ -0,0 +1,54 @@
+---
+vas:
+  uni04delta:
+    stages:
+      - path: examples/dt/uni04delta/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni04delta/control-plane
+        wait_conditions:
+          - >-
+            oc -n openstack wait osctlplane controlplane
+            --for condition=Ready
+            --timeout=600s
+        values:
+          - name: network-values
+        build_output: control-plane.yaml
+
+      - path: examples/dt/uni04delta/edpm-pre-ceph
+        wait_conditions:
+          - >-
+            oc -n openstack wait
+            osdpd edpm-deployment-pre-ceph --for condition=Ready
+            --timeout=1500s
+        values:
+          - name: edpm-values
+            src_file: values.yaml
+        build_output: dataplane-pre-ceph.yaml
+        post_stage_run:
+          - name: Deploy Ceph
+            type: playbook
+            source: "../../playbooks/ceph.yml"
+            inventory: "${HOME}/ci-framework-data/artifacts/zuul_inventory.yml"
+
+      - path: examples/dt/uni04delta
+        wait_conditions:
+          - >-
+            oc -n openstack wait osdpd
+            edpm-deployment
+            --for condition=Ready
+            --timeout=1200s
```

Comment:
the timeout here was not sufficient it took me around 22 minutes, so better to move it to 1800 or 2400 for slow systems

Reply:
Fixed